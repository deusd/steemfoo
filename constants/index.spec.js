import * as constants from './'

describe('constants', () => {
  it('should be available', () => {
    expect(constants).toMatchSnapshot()
  })
})
