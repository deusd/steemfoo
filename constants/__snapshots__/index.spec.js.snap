// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constants should be available 1`] = `
Object {
  "default": Object {},
}
`;
